lib/assessment/label.py

```diff
@@ -358,15 +357,22 @@ def compute_messages(self, prompt, rubric, student_code, examples=[]):
         return messages
 
     def get_response_data_if_valid(self, choice, rubric, student_id, choice_index=None, reraise=False, response_type='tsv'):
-        response_text = choice['message']['content']
+        response_text = None
+        finish_reason = None
+        if 'message' in choice.keys(): # OpenAI response
+            response_text = choice['message']['content']
+            finish_reason = choice['finish_reason']
+        elif 'completion' in choice.keys(): # Claude response
+            response_text = choice['completion']
+            finish_reason = choice['stop_reason']
```

Comment:
can you please clarify, did you use claude as part of verifying the CFE results and / or generating the new confidence levels? I assume the new confidence levels in the new release in S3 were generated by the exact params in that same release, but I'm just double checking to understand why we need Claude as part of a CFE PR

Reply:
I was using Claude as a stopgap, since I couldn't access GPT due to the credits issue we were having. I can re-run this with GPT now that we've dealt with that.

Reply:
Thanks Mark, since changes to confidence levels are a main way of catching regressions right now I think it would be best to rerun with GPT.